src/omnisharp/server.ts

```diff
@@ -259,6 +268,25 @@ export class OmniSharpServer {
             args.push('--debug');
         }
 
+        let launchPath: string;
+        if (this._options.path) {
+            try {
+                let serverUrl = "https://roslynomnisharp.blob.core.windows.net";
```

Comment:
Maybe move these into some obvious constant definitions somewhere?

Reply:
Where do you suggest can these be moved? Should we define some object that contains these fields in the package.json and just read from it here ?

Reply:
You could just move them up to be const fields in server.ts